src/client/driver/driver.js

```diff
@@ -128,8 +159,29 @@ export default class Driver {
         }
     }
 
+    _addUnexpectedDialogErrorToStatus (status) {
+        if (!status.isCommandResult)
+            return;
+
+        var dialogError = this._getUnexpectedDialogError();
+
+        if (!status.executionError)
```

Comment:
I suppose we should check `status.pageError` here (instead of `executionError`) to don't override an error that was appeared before